tests/pytorch/test_onnx_export.py

```diff
@@ -81,8 +93,9 @@ def do_export(
     fname: str,
     use_fp8: bool=True,
     opset: int=OPSET,
-    input_names: list=["input"],
-    output_names: list=["output"],
+    input_names: List=["input"],
+    output_names: List=["output"],
```

Comment:
Could we change these to `None` to have immutable default values just a good practice? This is for throughout the file, just making one comment.

Reply:
Of course - thanks for point this out!